projects/file-parsing/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	jsonparser "github.com/CodeYourFuture/immersive-go-course/projects/file-parsing/fileparser"
+)
+
+func getHighScorePlayer(players jsonparser.Players) string {
+	highScore := 0
+	highScorePlayer := ""
+	for _, player := range players {
+		if player.HighScore > highScore {
```

Comment:
What assumptions are you making about scores in this code?

Reply:
scores are positive numbers , which is wrong -_-